content/admin/Administration Panel/Roles and Security Groups.adoc

```diff
@@ -38,7 +38,7 @@ use logic hooks, workflow, or do a direct database insert into the
 securitygroups_records table if doing a one-time initial setup.
 
 {{%notice info%}}Further advanced security group settings and functionality are available with 
-the purchase of the https://store.suitecrm.com/addons/securitysuite[SecuritySuite^] 
+the purchase of the https://store.suitecrm.com/addons/securitysuite[SecuritySuite] 
```

Comment:
These `^` characters were there on purpose :-) They make the link open in a new window, which is appropriate for external sites.